### PR TITLE
Feature/open media file

### PR DESCRIPTION
**Pull request by Irmy**
Removed unnecessary boiler plate code regarding ipcrenderer.on
Connected import UI to renderer function and main listener. 
Linked media import to new project action.

### CODE REVIEW

**Comment by chloebrett** [comment]
Irmy and I paired on this, so will wait on approval from someone else

File: src/main/preload.js
Comment (chloebrett) [inline src/main/preload.js]: my bad, should be "don't expose whole IPC module"

File: src/renderer/components/NewProjectModal.tsx
Comment (MutazAshhab) [inline src/renderer/components/NewProjectModal.tsx]: In this file these imports are not being used: AudioFileExtension MediaFileExtension VideoFileExtension extractFileExtension getMediaType

**Review by chloebrett — APPROVED** [review]

**Review by MutazAshhab — APPROVED** [review]

**Pull request merged**
